recipes/conda-tree/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "conda-tree" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rvalieris/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0a2adacc7416d02b6a8a923bc23762655d0c28dd4f0498f3cd72ba3bffc4a1e7
+
+build:
+  number: 0
+  noarch: python
+  script:    # See https://github.com/conda/conda-build/issues/3166
+    - mkdir -p "$PREFIX/bin"
+    - cp conda-tree.py "$PREFIX/bin/conda-tree"
+
+requirements:
+  host:
+    - python
+  run:
+    - networkx
```

Comment:
I'd add in python as a run time dep as well

Reply:
I'll happily do that. I had incorrectly thought that packages in host were automatically in run.